src/modules/fullpageos/start_chroot_script

```diff
@@ -13,6 +13,16 @@ unpack /filesystem/home/pi /home/pi pi
 unpack /filesystem/boot /boot
 unpack /filesystem/root_init /
 
+FULLPAGEOS_BOOT_FILES=(
+"fullpageos.txt"
+"fullpagedashboard.txt"
+"check_for_httpd"
+)
+for FILE in "${FULLPAGEOS_BOOT_FILES[@]}"
+do
+    [ ! -f "/boot/${FILE}" ] && ln -s "/boot/firmware/${FILE}" "/boot/${FILE}"
```

Comment:
This one check if file (strong type = file) exist, symlink return false in this case.
It can be changed to:
```suggestion
    [ ! -e "/boot/${FILE}" ] && ln -s "/boot/firmware/${FILE}" "/boot/${FILE}"
```
but in case script will run only once at startup — maybe it doesn't matter